packages/ui-demo/src/SplitPage.stories.tsx

```diff
@@ -124,18 +124,20 @@ const ScrollableContent = () => {
   }));
 
   return (
-    <FlatList
-      contentContainerStyle={{height: "100%"}}
-      data={items}
-      keyExtractor={(item) => item.name}
-      renderItem={(item) => {
-        return (
-          <Box key={item.item.name} color="purple" padding={2}>
-            <Text>name: {item.item.name}</Text>
-          </Box>
-        );
-      }}
-    />
+    <ScrollView>
```

Comment:
I'm confused what this is testing, with a FlatList inside a full height scrollview. You generally would want to avoid doing that. You could do one child as a flat list and one as a scrollview with a bunch of content as a good test though (in the multiple children one?)

Reply:
Yea so wrapping the flat list in the scroll view is actually how I get the scroll to even work. If I remove it, the scroll stops working. I think it has to do with a flatlist/scrollview within the swiper library. The swiper library is actually built off of flatlist. The reason this is a test is because the content in flourish is all scrollable with the exception of chat view. I just want to make sure that children are still scrollable but I don't want to add another embedded scroll view when rendering the child if it's not required. The parent component can add a scrollview if the child isn't scrolling as intended. 

I hope that makes sense but definitely down to talk this through with you if that came out super confusing lol. Especially since you might have a cleaner solution

Reply:
That seems very odd, but hey, it works! I'm ok merging as-is and we can revisit later or if it becomes an issue.